config/samples/ironic_v1beta1_ironic.yaml

```diff
@@ -1,16 +1,9 @@
 apiVersion: ironic.openstack.org/v1beta1
 kind: Ironic
 metadata:
-  name: ironic
-  namespace: openstack
+  name: ironic-sample
```

Comment:
the kuttl test is failing because this name has changed and the asserts are looking for "ironic". I'll look around other operators to see what the convention is, and either update the kuttl asserts or revert the names here